packages/core/src/extensions/helpers/formatKeyboardShortcut.ts

```diff
@@ -1,7 +1,10 @@
-import browser from "../../lib/atlaskit/browser";
+const isAppleOS = () =>
```

Comment:
Move to util file (same comment as above)

Or, alternatively, keep the old "browser" file from atlaskit (just move it somewhere else)

Reply:
I'd rather not keep the whole browser file since we were only using it to check if `cmd` or `ctrl` should be used for keyboard shortcuts, so I'll probably just move that function to a utils file.